interfaces.go

```diff
@@ -22,8 +22,8 @@ import (
 	"errors"
 	"math/big"
 
```

Comment:
The main package will be still `ethereum`. Can we also rename it to `celo` (line 18)?

Reply:
Yep

Reply:
I'm less enthusiastic about this change.  The resulting conflicts in code when merging form upstream will not be trivial like ones from imports (and code changes more often than imports).  Would leaving the alias as ethereum cause any problems?  After all, any other project that imports from celo-blockchain can alias it to celo themselves.

Reply:
Ok, we're renaming it externally as `celo`, but keeping it as `ethereum` internally. Practically the internal rename should have no effect on other users.